jsonschema-core.md

```diff
@@ -523,11 +523,23 @@ to inform the user of the evaluation path that produced the error or annotation.
 
 ### Keyword Interactions
 
-Keyword behavior MAY be defined in terms of the annotation results of
-[subschemas](#root) and/or adjacent keywords (keywords within the same schema
-object) and their subschemas. Such keywords MUST NOT result in a circular
-dependency. Keywords MAY modify their behavior based on the presence or absence
-of another keyword in the same [schema object](#schema-document).
+Keywords MAY modify their behavior based on the presence or absence of another
+keyword or its value in the same [schema object](#schema-document). Such
```

Comment:
I'm finding this wording awkward. I suggest something more like this ...

```suggestion
Keywords MAY modify their behavior based on the presence, absence, or value of 
another keyword in the same [schema object](#schema-document). Such
```

Reply:
Yeah, it's awkward, but I think it's more correct.  It's:

- the presence or absence of the keyword
- the keyword's value

I feel they're separate concepts and so need to be listed separately.

Reply:
I disagree. Listing them together makes sense to me. 

* the presence of another keyword
* the absence of another keyword
* the value of another keyword

These things all seem to fit and make sense together.

Reply:
For me personally, I feel like these are all attributes of a keyword, so a single comma separated list feels more correct and easier to read.
But, I don't see that it matters a great deal, and isn't a blocker for merging this PR.